pkgs/development/python-modules/black-macchiato/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pytestCheckHook,
+  black
+}:
+
+buildPythonPackage rec {
+  pname = "black-macchiato";
+  version = "1.3.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner  = "wbolster";
+    repo   = pname;
+    rev    = "version";
```

Comment:
```suggestion
    rev    = version;
```

Reply:
since version is a string to begin with, we don't have to interpolate it, we can just pass the version variable to rev

Reply:
Ah, my mistake – I should have caught that! Fixing.